projects/demo-integrations/cypress/tests/desktop/kit/input-count/input-count.cy.ts

```diff
@@ -0,0 +1,43 @@
+import {WAIT_BEFORE_SCREENSHOT} from '@demo-integrations/support/properties/shared.entities';
+
+describe(`InputCount`, () => {
+    for (const empty of [false, true]) {
+        it(`empty=${empty}`, () => {
+            cy.tuiVisit(`components/input-count/API?empty=${empty}`);
+
+            cy.get(`#demo-content`)
+                .wait(WAIT_BEFORE_SCREENSHOT)
+                .matchImageSnapshot(`01-input-count-empty-${empty}`);
+
+            cy.get(`#demo-content`)
+                .findByAutomationId(`tui-primitive-textfield__native-input`)
+                .focus()
+                .type(`{selectall}{backspace}`)
+                .blur();
+
+            cy.get(`#demo-content`)
+                .wait(WAIT_BEFORE_SCREENSHOT)
+                .matchImageSnapshot(`02-input-count-empty-${empty}`);
+
+            cy.get(`#demo-content`)
+                .findByAutomationId(`tui-primitive-textfield__native-input`)
+                .focus()
+                .type(`1234`)
+                .blur();
+
+            cy.get(`#demo-content`)
+                .wait(WAIT_BEFORE_SCREENSHOT)
```

Comment:
It generates this screenshot
![input-count:input-count cy ts:__diff_output__:02-input-count-empty-true diff](https://user-images.githubusercontent.com/35179038/209639086-aae8d0a8-99bb-47fd-9e06-2c99c1024aee.png)

Reply:
<img width="1057" alt="Screenshot 2022-12-27 at 11 46 33" src="https://user-images.githubusercontent.com/35179038/209639546-d5f107a8-1936-4c56-99b2-9d25f6391d1c.png">

Reply:
`It generates this screenshot` expected

Reply:
<img width="344" alt="image" src="https://user-images.githubusercontent.com/12021443/209685179-08361423-ad68-48a8-82de-6a4dab73996c.png">
fixed